lldb/tools/lldb-dap/DAP.h

```diff
@@ -64,6 +65,70 @@ typedef llvm::DenseMap<lldb::addr_t, InstructionBreakpoint>
 using AdapterFeature = protocol::AdapterFeature;
 using ClientFeature = protocol::ClientFeature;
 
+/// Global DAP session manager
+class DAPSessionManager {
```

Comment:
Can you move this class into its own `DAPSessionManager.h/DAPSessionManager.cpp`?